src/components/Home/Home.jsx

```diff
@@ -57,4 +51,6 @@ export default function Home({ userName, setSelectedEncounter }) {
 };
 
 Home.propTypes = {
+  userName: PropTypes.string,
```

Comment:
Should we have `isRequired` on this one, since a string _must_ be passed in the query to get the user's encounters?

Reply:
I tried it with `isRequired` and it threw some errors due to it starting as `null`, I can try changing it to an empty string instead of null if we do not have anything dependent on the falsy value of null.

Reply:
That sounds fair to me! Let's make a note of it to look into it during p2. 👍